prow/cmd/checkconfig/main.go

```diff
@@ -473,3 +514,21 @@ func validateNeedsOkToTestLabel(cfg *config.Config) error {
 	}
 	return errorutil.NewAggregate(queryErrors...)
 }
+
+func verifyOwnersPlugin(cfg *plugins.Configuration) error {
+	// we do not know the set of repos that use OWNERS, but we
+	// can get a reasonable proxy for this by looking at where
+	// the `approve', `blunderbuss' and `owners-label' plugins
+	// are enabled
+	approveConfig := enabledOrgReposForPlugin(cfg, approve.PluginName)
+	blunderbussConfig := enabledOrgReposForPlugin(cfg, blunderbuss.PluginName)
+	ownersLabelConfig := enabledOrgReposForPlugin(cfg, ownerslabel.PluginName)
+	ownersConfig := approveConfig.union(blunderbussConfig).union(ownersLabelConfig)
+	validateOwnersConfig := enabledOrgReposForPlugin(cfg, verifyowners.PluginName)
+
+	invalid := validateOwnersConfig.difference(ownersConfig).items()
```

Comment:
This difference is backwards. It should be `ownersConfig.difference(validateOwnersConfig)`.